### PR TITLE
Add notify reward script for gnosis, optimism, zkevm sep 2024

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to  https://github.com/Giveth/giveth-dapps-v2/issues/4434

**ZKEVM Mainnet Config** https://www.notion.so/giveth/641e512e33124a249177f67ba4762044?v=86ba47af2a9a4927a68cf0a8fcbe5c7d

### CODE REVIEW

File: deployments/notifyRewardAmount/optimism/givpower_distribute_extended_sep_2024.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    );
+    const periodFinish = await unipoolTokenDistributor.periodFinish();
+    const duration = await unipoolTokenDistributor.duration();
+
+    // 10 minutes of precision
+    if (periodFinish < currentTime + 60 * 10) {
+        const pos = Math.floor((currentTime - initTime) / duration);
+        console.log("pos:", pos);
+        if (pos < 0) return;
+        if (distro[pos] === 0) return;
```
Comment (aminlatifi) [inline deployments/notifyRewardAmount/optimism/givpower_distribute_extended_sep_2024.js:48]: What is this for?! in purpose?

Comment (mohammadranjbarz) [reply deployments/notifyRewardAmount/optimism/givpower_distribute_extended_sep_2024.js:48]: > What is this for?! in purpose?

I just used the last notifyReward script and changed the params and addresses, .. , didn't change the code logic, WDYT should we delete it?

Comment (aminlatifi) [reply deployments/notifyRewardAmount/optimism/givpower_distribute_extended_sep_2024.js:48]: We may have put that condition in place due to some special situation we had with optimism. If the current OP rewarding program ends around the the start of the new program, we don't need that condition.

**Comment by aminlatifi** [comment]
@mohammadranjbarz Don't forget to set the reward notifier on the zkevm unipool.

**Review by aminlatifi — APPROVED** [review]
LGTM, thanks @mohammadranjbarz

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz Don't forget to set the reward notifier on the zkevm unipool.
@aminlatifi 
I just changed the reward distribution of polygon zkevm unipool givPower https://zkevm.polygonscan.com/address/0xc790f82bf6f8709aa4a56dc11afad7af7c2a9867#readProxyContract to what is the reward distribution for unipool givPower on gnosis smart contract https://gnosisscan.io/address/0xD93d3bDBa18ebcB3317a57119ea44ed2Cf41C2F2#readProxyContract

FYI @laurenluz

**Pull request merged**